index.html

```diff
@@ -69,5 +69,167 @@ <h1>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+     <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+     <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+     <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+    </section>
+    <section class="aboutMe headline">
+      <h2 >About me</h2>
+      <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+      <button class="seeProject inter-font">Get My Resume</button>
+      <ul class="skills">
+        <li>
+          <img src="/assets/images/language.svg" alt="language Image ">
+          <a href="#" class="inter-font">Language</a>
+          <ul class="SkillsBTNs inter-font">
+            <li>
+              <button>Javascript</button>
+            </li>
+            <li>
+              <button>Ruby</button>
+            </li>
+            <li>
+              <button>Html</button>
+            </li>
+            <li>
+              <button>CSS</button>
+            </li>
+          </ul>
+        </li>
+      </ul>
+      <ul class="skills">
+        <li>
+          <img src="/assets/images/frameworks.svg" alt="Frameworks Image ">
```

Comment:
- [ ] As suggested by the previous reviewer, I kindly suggest that you fix buttons overflowing horizontally. please adjust them
<img width="383" alt="image" src="https://user-images.githubusercontent.com/52934128/185573327-162869f6-c4e0-4e35-88f4-0c3cf185e7f0.png">